english/got/got.md

```diff
@@ -0,0 +1,2 @@
+Who's excited about Game of Thrones Season 8?
+Then head [here](https://github.com/bhavesh-dhera/create-your-own-adventure/blob/master/english/marshmallow.md) and continue your work!
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.